libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -742,8 +742,7 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			{
 				string vars = IRVariable(arg).commaSeparatedList();
```

Comment:
@ekpyron should `IRVariable` provide public access to the `stackSlots` function? Then we can just a vector of strings and then pass it through `joinHumanReadable` at the point where we send it to the template.

Reply:
Yeah, making ``IRVariable::stackSlots`` public makes sense!